qiskit/quantum_info/states/cython/exp_value.pyx

```diff
@@ -0,0 +1,55 @@
+#!python
+#cython: language_level = 3
+#distutils: language = c++
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+cimport cython
+import numpy as np
+
+cpdef unsigned long long popcount(unsigned long long count):
```

Comment:
You are not calling this from Python so should use `cdef` only to avoid a 10% performance penalty.